packages/internal/generated-clients/src/imx/base.ts

```diff
@@ -18,7 +18,7 @@ import { Configuration } from "./configuration";
 // @ts-ignore
 import globalAxios, { AxiosPromise, AxiosInstance, AxiosRequestConfig } from 'axios';
 
-export const BASE_PATH = "https://api.sandbox.x.immutable.com".replace(/\/+$/, "");
+export const BASE_PATH = "https://api.sandbox.immutable.com".replace(/\/+$/, "");
```

Comment:
why no `x` anymore? 
all our docs has the `x` in the example queries.

Reply:
I tried hunting down the source of this change, it's a bit of a mystery why the `x` is removed since the `host` in the spec file contains the `x`. API requests to the endpoints without the `x` do work though, I've tested several endpoints with and without the `x` and they both perform the same.

Reply:
This _should_ work, but i cannot guarantee it @CodeSchwert @wdinau . We put both hosts in for the eventual migration to this hostname for "multil rollup".
In saying this, it is 'cleaner' for now (maybe?) for customers to understand .x relates to different API functionality?

Reply:
Opted to go with the "cleaner" solution. Reverted the URL to include the `x`.

We should probably leave the URL change to a seperate epic / PR.